### ISSUE TITLE
Facet Filters Accessibility

### ISSUE THREAD

**Issue by mcquinne** [cedardevs/onestop#234]
Within the facet filters section of the filter menu, you should be able to navigate/select the filters you want to apply via screen reader/keyboard.

Part of epic #142

**Comment by zebdelk** [comment]
See #233 for more context.

Relative structure:
```
<h3>Keywords</h3>
<a>Show/Hide Keyword Filters</a>
<!-- The following elements are invisible when hidden -->
<h4>Data Theme</h4>
<ul>
<li>group1 <a>expand group1</a></li>
<li>group2 <a>collapse group2</a>
  <ul>
    <li>subgroup1</li>
    <li>subgroup2</li>
  </ul>
</li>
</ul>
<a>Show/Hide Data Theme Keyword Filters</a>
<h4>Platform</h4>
<a>Show/Hide Platform Keyword Filters</a>
<ul>
<li>checkbox1</li>
<li>checkbox2</li>
</ul>
<h4>Projects</h4>
<a>Show/Hide Projects Keyword Filters</a>
...
```

- [ ] Sections have appropriate headers
- [ ] Use links styled to look like buttons
- [ ] (TBD) Show/Hide links notify screen readers
- [ ] Link behavior is clear to screen reader users (links may be navigated out of context)
- [ ] Links names are not too long / over-descriptive
- [ ] Checkbox / form labels are used correctly to read out keywords
- [ ] Checkbox / form labels are concise, but clear in non-linear contexts
- [ ] (TBD) Filter Section behavior is consistent (apply filter on change vs apply button)
